RELEASE_STEPS.md

```diff
@@ -0,0 +1,54 @@
+# Release Notes publishing checklist
+
+- Add the release note text inside `_includes/releases` folder as a markdown file, without headers.
+  As the contents of these files are raw, they can be checkout from external sources (eg, once the docs have 
```

Comment:
s/check/checked/

Reply:
How about the following for lines 4 and 5:

As the contents of these files are raw, they can be checked out from external sources (once the docs have been moved to the extensions repository).